pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -602,6 +602,9 @@ var _ = Describe("Pod Network", func() {
 	})
 
 	Context("Masquerade startDHCP", func() {
+		expectedSocketPath := fmt.Sprintf("/proc/self/root/var/run/kubevirt/sockets/%s", api.DefaultBridgeName)
+		maxRetryNumber := 5
+
 		It("should succeed when DHCP server started", func() {
```

Comment:
s/should succeed when DHCP server started/should succeed when DHCP and RA servers started